services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/ilmoitus/Ilmoitus.kt

```diff
@@ -0,0 +1,14 @@
+package fi.hel.haitaton.hanke.ilmoitus
+
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class Ilmoitus(
```

Comment:
Yes, the naming could be better. "Ilmoitus" could mean a dozen of different things. It could use a prefix to clarify the meaning. "Valmistumisilmoitus" might be one candidate. Let's talk with the Allu domain expert to see if they have any ideas.

Reply:
Would it help if `ilmoitus` (and `paatos` and maybe even `allu` as well) packages were under `hakemus` package?

Reply:
I don't think moving it under hakemus would help. Several concepts named ilmoitus could be in the hakemus package as well. E.g. kaivuilmoitus and muutosilmoitus.

I'm thinking Valmistumisilmoitus would be correct. It's distinct from muutosilmoitus and kaivuilmoitus. It might conflict with an email about the application moving to TYO_VALMIS state, but I think there's not much confusion there.

Reply:
Sounds good.